scss/forms/_form-control.scss

```diff
@@ -192,6 +192,26 @@ textarea {
   width: $form-color-width;
   height: $input-height;
   padding: $input-padding-y;
+  border-color: var(--#{$prefix}body-color);
+
+  // Boosted mod
+  &:not(:disabled):hover {
+    background-color: var(--#{$prefix}body-color);
+  }
+
+  &:disabled {
+    background-color: var(--#{$prefix}body-bg);
+    border-color: $form-color-disabled-color;
+
+    &::-moz-color-swatch {
+      filter: brightness(0) invert(1) brightness(.8);
+    }
+
+    &::-webkit-color-swatch {
+      filter: brightness(0) invert(1) brightness(.8);
```

Comment:
Should we introduce Sass variables everywhere for a more customizable color picker ?